packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1540,9 +1540,11 @@
   "invisible": "invisible",
   "Invisible": "Invisible",
   "Invitation": "Invitation",
-  "Invitation_Email_Description": "You may use the following placeholders: <br/><ul><li>[email] for the recipient email.</li><li>[Site_Name] and [Site_URL] for the Application Name and URL respectively.</li></ul>",
+  "Invitation_Email_Default":"You have been invited to [Invite_Link]",
```

Comment:
This string ends up in the title of the email, and not the body.  You can't click the [Invite_Link] from the email title, it needs to be in the body.  Perhaps this should include the app name instead..... "You have been invited to join Viasat Connect"